src/algorithms.jl

```diff
@@ -1211,6 +1211,27 @@ pages={113753}
 """
 struct ERKN7 <: OrdinaryDiffEqAdaptivePartitionedAlgorithm end
 
+struct RKN4 <: OrdinaryDiffEqAlgorithm end
+
+"""
```

Comment:
Docstrings go above what they are documenting.

Reply:
just fixed it